morecantile/data/WGS1984Quad.json

```diff
@@ -1,255 +1,325 @@
-        {
-            "type": "TileMatrixType",
-            "identifier": "10",
-            "scaleDenominator": 272989.386732772,
-            "topLeftCorner": [
-                90,
-                -180
-            ],
-            "tileWidth": 256,
-            "tileHeight": 256,
-            "matrixWidth": 2048,
-            "matrixHeight": 1024
-        },
-        {
-            "type": "TileMatrixType",
-            "identifier": "11",
-            "scaleDenominator": 136494.693366386,
-            "topLeftCorner": [
-                90,
-                -180
-            ],
-            "tileWidth": 256,
-            "tileHeight": 256,
-            "matrixWidth": 4096,
-            "matrixHeight": 2048
-        },
-        {
-            "type": "TileMatrixType",
-            "identifier": "12",
-            "scaleDenominator": 68247.346683193,
-            "topLeftCorner": [
-                90,
-                -180
-            ],
-            "tileWidth": 256,
-            "tileHeight": 256,
-            "matrixWidth": 8192,
-            "matrixHeight": 4096
-        },
-        {
-            "type": "TileMatrixType",
-            "identifier": "13",
-            "scaleDenominator": 34123.6733415964,
-            "topLeftCorner": [
-                90,
-                -180
-            ],
-            "tileWidth": 256,
-            "tileHeight": 256,
-            "matrixWidth": 16384,
-            "matrixHeight": 8192
-        },
-        {
-            "type": "TileMatrixType",
-            "identifier": "14",
-            "scaleDenominator": 17061.8366707982,
-            "topLeftCorner": [
-                90,
-                -180
-            ],
-            "tileWidth": 256,
-            "tileHeight": 256,
-            "matrixWidth": 32768,
-            "matrixHeight": 16384
-        },
-        {
-            "type": "TileMatrixType",
-            "identifier": "15",
-            "scaleDenominator": 8530.91833539913,
-            "topLeftCorner": [
-                90,
-                -180
-            ],
-            "tileWidth": 256,
-            "tileHeight": 256,
-            "matrixWidth": 65536,
-            "matrixHeight": 32768
-        },
-        {
-            "type": "TileMatrixType",
-            "identifier": "16",
-            "scaleDenominator": 4265.45916769956,
-            "topLeftCorner": [
-                90,
-                -180
-            ],
-            "tileWidth": 256,
-            "tileHeight": 256,
-            "matrixWidth": 131072,
-            "matrixHeight": 65536
-        },
-        {
-            "type": "TileMatrixType",
-            "identifier": "17",
-            "scaleDenominator": 2132.72958384978,
-            "topLeftCorner": [
-                90,
-                -180
-            ],
-            "tileWidth": 256,
-            "tileHeight": 256,
-            "matrixWidth": 262144,
-            "matrixHeight": 131072
-        }
-    ]
-}
+  "id": "WorldCRS84Quad",
```

Comment:
@dchirst sorry to just catch this now, but any good reason for this to go from `WGS1984Quad` to `WorldCRS84Quad`? 

I believe the ID should be `WGS1984Quad`. The URI is fine being `http://www.opengis.net/def/tilematrixset/OGC/1.0/WorldCRS84Quad` as discussed in
ref: https://github.com/opengeospatial/2D-Tile-Matrix-Set/issues/60